websites/W/Wolfy/presence.ts

```diff
@@ -178,35 +190,59 @@ presence.on("UpdateData", async () => {
 			currTime = document.querySelector("div.Header_timer__36MsP").textContent;
 		}
 
+		presenceData.details = "En jeu";
+
+		presenceData.smallImageKey = "live";
+		presenceData.smallImageText = "En jeu";
+		if (currTime?.includes(":")) {
+			[presenceData.startTimestamp, presenceData.endTimestamp] = getTimestamps(
```

Comment:
you could probably use presence.getTimestampsFromFormat here but I'm not entirely sure, you should test that

Reply:
It was from the original code made by @fr0r, i didn't really looked deeper about this and didn't changed anything

Reply:
well then why is this in the diff?

Reply:
Just moved it before the handling of the state "WAITING" in game to set image and time before handling specific game's state, there is no impact on running

Reply:
then why did you move it?

Reply:
Just for code visibility to assign general things before specefic things.

Can revert it if you want

Reply:
If this has no practical impact please do

Reply:
Revert done